main/display/display.c

```diff
@@ -25,7 +25,7 @@
 // Constant data
 //==============================================================================
 
-const uint32_t sin1024[] =
+const int16_t sin1024[] =
```

Comment:
You only need a quarter of this table if you want to do the symmetry thing here too